packages/block-editor/src/components/leaf-more-menu/index.js

```diff
@@ -17,7 +17,7 @@ import { __, sprintf } from '@wordpress/i18n';
  */
 import { store as blockEditorStore } from '../../store';
 import BlockTitle from '../block-title';
-import { useListViewContext } from './context';
+import { useListViewContext } from '../list-view/context';
```

Comment:
This context is in `list-view`. Since this component is no longer within `list-view`, should we pass the `expandedState` and `expanded` function in rather than `useListViewContext`? Or is it OK to use the context like this across components?

Reply:
As things are I'd probably suggest passing the state manually rather than relying on context here. This isn't really a global context but one that's local to list view.

That said will Leaf ever exist outside of a List View? If not then perhaps it needs to be colocated with List View?

Reply:
@scruffian what do you think? I kind of agree with Dave here, I can't think of another place that would use the component besides list view

Reply:
I think that this component should live in the navigation block itself.